docs/docs/reference/ui/atoms/Badge.mdx

```diff
@@ -23,3 +41,17 @@ import { Badge } from '@faststore/ui'
 ## Customization
 
 `data-store-badge`
+
+## Best practices
+
+### Do's
+- Write clear and short texts.
+- Place the badge next to or on top of the product related to the badge.
+- Use a color palette to establish the relevance and status of the product and get the users' attention.
+
+### Don'ts
+- Avoid placing badges next to buttons. Although they look similar, a badge is not as interactive as a button.
```

Comment:
```suggestion
### Do's

- Place the badge next to or on top of its related item.
- Be cautious about the badge's positioning, writing, shape, and color to avoid any confusion with components similar in appearance.

### Don'ts

- Don't use long texts in badges. A reasonable character limit is 15.
- Don't label badges with active verbs.
- Don't use color alone to convey information. If color is needed to convey meaning, ensure there is a text alternative for that meaning as well.
- Don't place badges next to buttons. Although badges and buttons are similar in appearance, badges are not interactive and may confuse users if not used in the proper context.
```